### ISSUE TITLE
✅ 🐞 Bug Report: Nested Carousel makes it very difficult to navigate through slides on mobile - [ED-11140]

### ISSUE THREAD

**Issue by Michaelgimii** [elementor/elementor#22827]
### Prerequisites

- [X] I have searched for similar issues in both open and closed tickets and cannot find a duplicate.
- [X] The issue still exists against the latest stable version of Elementor.

### Description

Hi elementor team.

After the update, it is very difficult to scroll through the slides in the carousel widget. In fact, when the user is viewing the website on mobile, she cannot easily scroll through the carousel without using the arrow buttons. I am experiencing this problem in Chrome browser - Android.

### Steps to reproduce

1- Add a new carousel widget on the page and place an image widget inside the two slides.
2- View the website on mobile (Chrome browser - Android).
3- Try to navigate the carousel without clicking on the arrow icons.

It will be very difficult.

Hi@nicholaszein, If you are not successful in reproducing, please do not lock this issue so that I can further investigate the cause of the problem. I hope you succeed in reproducing it in the first step.

### Isolating the problem

- [X] This bug happens with only Elementor plugin active (and Elementor Pro).
- [X] This bug happens with a Blank WordPress theme active ([Hello theme](https://wordpress.org/themes/hello-elementor/)).
- [X] I can reproduce this bug consistently following the steps above.

### System Info

<details>
<summary>System Info</summary>

```txt
					
== Server Environment ==
	Operating System: WINNT
	Software: Apache/2.4.38 (Win32) OpenSSL/1.1.1a PHP/7.3.2
	MySQL version: mariadb.org binary distribution v5.6.42-84.2
	PHP Version: 7.4
	PHP Memory Limit: 128M
	PHP Max Input Vars: 1000
	PHP Max Post Size: 64M
	GD Installed: Yes
	ZIP Installed: Yes
	Write Permissions: All right
	Elementor Library: Connected

== WordPress Environment ==
	Version: 6.2.2
	Site URL: http://domain.com
	Home URL: http://domain.com
	WP Multisite: No
	Max Upload Size: 64 MB
	Memory limit: 40M
	Max Memory limit: 256M
	Permalink Structure: /%year%/%monthnum%/%day%/%postname%/
	Language: en-US
	Timezone: Europe/Berlin
	Admin Email: [REDACTED]
	Debug Mode: Inactive

== Theme ==
	Name: Hello Elementor
	Version: 2.7.1
	Author: Elementor Team
	Child Theme: No

== User ==
	Role: administrator
	WP Profile lang: en_US
	User Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/114.0.0.0 Safari/537.36

== Active Plugins ==
	Elementor
		Version: 3.14.0
		Author: Elementor.com

	Elementor Pro
		Version: 3.14.0
		Author: Elementor.com


== Features ==
	Custom Fonts: 4
	Custom Icons: 0

== Integrations ==
	


== Elementor Experiments ==
	Optimized DOM Output: Active
	Improved Asset Loading: Inactive
	Improved CSS Loading: Active
	Inline Font Icons: Inactive
	Accessibility Improvements: Inactive
	Additional Custom Breakpoints: Inactive
	admin_menu_rearrangement: Inactive by default
	Flexbox Container: Active
	Upgrade Swiper Library: Active
	Grid Container: Active
	Default to New Theme Builder: Inactive
	Hello Theme Header & Footer: Inactive
	Editor Top Bar: Active
	Landing Pages: Inactive
	Nested Elements: Active
	Nested Accordion: Inactive by default
	Lazy Load Background Images: Inactive
	Global Style Guide: Inactive by default
	Page Transitions: Inactive
	Notes: Inactive
	Loop: Active
	Form Submissions: Active by default
	Scroll Snap: Inactive
	Menu: Active


== Log ==
	
Log: showing 20 of 272023-05-12 02:53:23 [info] Elementor Pro/Upgrades - _on_each_version Start  
2023-05-12 02:53:23 [info] Elementor Pro/Upgrades - _on_each_version Finished 
2023-05-12 02:53:23 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.13.0',
  'to' => '3.13.1',
)]
2023-05-17 15:08:21 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor',
  'from' => '3.13.1',
  'to' => '3.13.2',
)]
2023-05-17 15:08:22 [info] elementor::elementor_updater Started 
2023-05-17 15:08:22 [info] Elementor/Upgrades - _on_each_version Start  
2023-05-17 15:08:22 [info] Elementor/Upgrades - _on_each_version Finished 
2023-05-17 15:08:22 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor',
  'from' => '3.13.1',
  'to' => '3.13.2',
)]
2023-05-17 15:08:24 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor',
  'from' => '3.13.1',
  'to' => '3.13.2',
)]
2023-06-19 18:54:50 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.13.1',
  'to' => '3.14.0',
)]
2023-06-19 18:54:53 [info] elementor-pro::elementor_pro_updater Started 
2023-06-19 18:54:53 [info] Elementor Pro/Upgrades - _on_each_version Start  
2023-06-19 18:54:53 [info] Elementor Pro/Upgrades - _on_each_version Finished 
2023-06-19 18:54:53 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.13.1',
  'to' => '3.14.0',
)]
2023-06-19 18:54:54 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.13.1',
  'to' => '3.14.0',
)]
2023-06-19 18:55:26 [info] elementor::elementor_updater Started 
2023-06-19 18:55:26 [info] Elementor/Upgrades - _on_each_version Start  
2023-06-19 18:55:26 [info] Elementor/Upgrades - _on_each_version Finished 
2023-06-19 18:55:26 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor',
  'from' => '3.13.2',
  'to' => '3.14.0',
)]
2023-06-19 18:55:27 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor',
  'from' => '3.13.2',
  'to' => '3.14.0',
)]

PHP: showing 5 of 5PHP: 2023-05-09 16:28:31 [error X 32][http://domain.com\wp-content\plugins\elementor\data\v2\manager.php::8] Maximum execution time of 30 seconds exceeded [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2023-05-11 01:51:51 [notice X 4][http://domain.com\wp-content\plugins\elementor\includes\managers\image.php::36] Undefined index: id [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2023-05-19 12:19:14 [notice X 25][http://domain.com\wp-content\plugins\elementor\core\common\modules\ajax\module.php::175] Undefined index: data [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2023-05-19 12:52:24 [notice X 3][http://domain.com\wp-content\plugins\elementor-pro\modules\theme-builder\classes\conditions-manager.php::171] Undefined index: editor_post_id [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2023-06-11 21:14:35 [notice X 9][http://domain.com\wp-content\plugins\elementor-pro\modules\woocommerce\module.php::381] Undefined offset: 1 [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]

JS: showing 10 of 10JS: 2023-05-19 07:47:47 [error X 6][http://domain.com/wp-content/plugins/elementor/assets/js/editor.min.js?ver=3.13.2:3:1097282] Cannot read properties of null (reading 'getBoundingClientRect') 
JS: 2023-05-19 08:28:21 [error X 5][http://domain.com/wp-content/plugins/elementor/assets/js/7fada1a61d72da6d46ab.bundle.min.js:2:7826] Cannot read properties of null (reading 'dataset') 
JS: 2023-05-27 22:07:09 [error X 1][http://domain.com/wp-content/plugins/elementor/assets/lib/backbone/backbone.marionette.min.js?ver=2.4.5.e1:24:19952] View (cid: "view3596") has already been destroyed and cannot be used. 
JS: 2023-05-29 13:06:40 [error X 1][http://domain.com/wp-content/plugins/elementor/assets/lib/backbone/backbone.marionette.min.js?ver=2.4.5.e1:24:19952] View (cid: "view1682") has already been destroyed and cannot be used. 
JS: 2023-05-29 13:20:42 [error X 3][http://domain.com/wp-content/plugins/elementor/assets/lib/backbone/backbone.marionette.min.js?ver=2.4.5.e1:24:19952] View (cid: "view1250") has already been destroyed and cannot be used. 
JS: 2023-06-01 13:21:06 [error X 1][http://domain.com/wp-content/plugins/elementor/assets/lib/backbone/backbone.marionette.min.js?ver=2.4.5.e1:24:19952] View (cid: "view1791") has already been destroyed and cannot be used. 
JS: 2023-06-01 22:27:06 [error X 1][http://domain.com/wp-content/plugins/elementor/assets/js/7fada1a61d72da6d46ab.bundle.min.js:2:8038] Cannot read properties of undefined (reading 'view') 
JS: 2023-06-04 10:03:41 [error X 6][http://domain.com/wp-admin/load-scripts.php?c=1&loadchunk_0=jquery-ui-draggable,react,react-dom,mce-view,imgareaselect&ver=6.2.2:29:23751] Cannot read properties of undefined (reading 'setValue') 
JS: 2023-06-11 14:54:44 [error X 1][http://domain.com/elementor/wp-content/plugins/elementor/assets/lib/backbone/backbone.marionette.min.js?ver=2.4.5.e1:24:19952] View (cid: "view6962") has already been destroyed and cannot be used. 
JS: 2023-06-11 14:56:46 [error X 1][http://domain.com/wp-content/plugins/elementor/assets/lib/backbone/backbone.marionette.min.js?ver=2.4.5.e1:24:19952] View (cid: "view10324") has already been destroyed and cannot be used. 



== Elementor - Compatibility Tag ==
	
	Elementor Pro: Compatible

== Elementor Pro - Compatibility Tag ==
	
```

<details>

**Comment by nicholaszein** [comment]
Hi @Michaelgimii! 

Thank you for submitting your issue! 🙏 

Apologies if I got it wrong, but you're reporting you can't swipe through slides on mobile. Is that it? Can you elaborate on what exactly is the issue? I've done some tests and it seems to work just fine for me. Please check the video below:

https://github.com/elementor/elementor/assets/113993460/42d590cc-a75c-4bea-ae01-b27b26128342

----

Please, let us know what is the problem in more detail so we can understand it and help you better.


Best regards

**Comment by Michaelgimii** [comment]
Hi @nicholaszein, I apologize for the delay.

In the above video, did you change the slide by clicking on the arrows, or by using a swipe?

**Comment by nicholaszein** [comment]
Hi @Michaelgimii, no worries! 😉

I swiped with my fingers.

**Comment by 79ho3ein** [comment]
Hi @nicholaszein,

I wanted to report the same problem but saw it was already reported here. I am also experiencing this problem and no matter what I did, I could not find the reason for this problem. I used the old carousels and they didn't have any problems, but the new carousel can't be scrolled at all on the mobile phone. Please see the link below. Do you feel a problem?

https://avangtheme.ir/order

**Comment by nicholaszein** [comment]
Hey @Michaelgimii and @79ho3ein!

Apologies for the delay and thanks for your patience! 🙏 

### This issue was resolved in Elementor Pro v3.14.1! 🥳 

✅ Feel free to check it out and update your plugin to the new version!

> _Check the changelog for more information:_
> [Changelog of Elementor and Elementor Pro](https://elementor.com/help/changelog-of-elementor/)

Cheers 🥂
